### DISCUSSION TITLE
Delete Workspace via API

### DISCUSSION THREAD

**Discussion by shinyshoes404** [coder/coder#12680] [General] [answered]
Is it possible to delete a workspace using the V2 API? I'm hoping I just missed something in the documentation.

I'm building an app to create users via SMS, and I want to delete the users after 12 hours (This is for a hands on coding demo/class). However, I can't delete users that have workspaces.

**Comment by shinyshoes404** [comment, marked as answer]
After poking around in the UI and network monitor, I finally figured it out.

__POST__   `/api/v2/workspaces/{workspace id}/builds`

__body__  
```
{
    "orphan": false,
    "transition": "delete"
}
```

**Reply by matifali** [reply]
Its documented [here](https://coder.com/docs/api/builds#create-workspace-build).
